### PR TITLE
[HOPS-1571] Partition hdfs_retry_cache table

### PR DESCRIPTION
**Pull request by smkniazi**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added and passed (for bug fixes / features)
- [ ] HOPS JIRA issue has been opened for this PR
- [ ] All commits have been squashed down to a single commit
- [ ] The commit message has the following format: [HOPS-XXX] message

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: src/main/java/io/hops/metadata/hdfs/entity/RetryCacheEntry.java [hunk 1]
```diff
@@ -18,48 +18,58 @@
 package io.hops.metadata.hdfs.entity;
 
 import io.hops.metadata.common.FinderType;
-import java.util.Arrays;
+import org.omg.PortableInterceptor.SUCCESSFUL;
```
Comment (berthoug) [inline src/main/java/io/hops/metadata/hdfs/entity/RetryCacheEntry.java:21]: Where is this import used?

File: src/main/java/io/hops/metadata/hdfs/entity/RetryCacheEntry.java [hunk 2]
```diff
@@ -18,48 +18,58 @@
...
 
+import java.util.Arrays;
+import java.util.Objects;
 
 public class RetryCacheEntry {
+
+  public static byte SUCCESS = (byte) 1;
+  public static byte FAILED = (byte) -1;
```
Comment (berthoug) [inline src/main/java/io/hops/metadata/hdfs/entity/RetryCacheEntry.java:29]: Is this used? Shouldn't it match or replace: https://github.com/hopshadoop/hops/blob/d2f89602e4a3b92501f392de1a40a7cf514311b4/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/RetryCache.java#L60

File: src/main/java/io/hops/metadata/hdfs/entity/RetryCacheEntry.java (hunk 1 above)
Comment (smkniazi) [reply src/main/java/io/hops/metadata/hdfs/entity/RetryCacheEntry.java:21]: fxd

File: src/main/java/io/hops/metadata/hdfs/entity/RetryCacheEntry.java (hunk 2 above)
Comment (smkniazi) [reply src/main/java/io/hops/metadata/hdfs/entity/RetryCacheEntry.java:29]: Its  fine. We have our own implementation so Values can differ

**Pull request merged**